primitives/src/asset.rs

```diff
@@ -53,6 +56,24 @@ pub enum Asset<MI: MaxEncodedLen> {
     ParimutuelShare(MI, CategoryIndex),
 }
 
+/// The `MarketAsset` enum represents all types of assets available in the
+/// prediction market protocol.
+///
+/// # Types
+///
+/// * `MI`: Market Id
+#[cfg_attr(feature = "std", derive(serde::Deserialize, serde::Serialize))]
+#[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
+#[derive(Clone, Copy, Debug, Decode, Default, Eq, Encode, MaxEncodedLen, PartialEq, TypeInfo)]
+pub enum PredictionMarketAsset<MI: MaxEncodedLen> {
+    CategoricalOutcome(MI, CategoryIndex),
+    #[default]
+    CombinatorialOutcome,
```

Comment:
This seems odd. I think `CombinatorialOutcome` shouldn't be in here at all, nor should it be the default.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1177/commits/e7c912cbeb61dee905da5a7009ff9c294007e0fd